plugins/lighthouse/src/api.ts

```diff
@@ -85,13 +85,32 @@ export interface Website {
 /** @public */
 export type WebsiteListResponse = LASListResponse<Website>;
 
+/** @public */
+export enum FormFactor {
+  Mobile = 'mobile',
+  Desktop = 'desktop',
+}
+
+/** @public */
+export type ScreenEmulation = {
+  mobile: boolean;
+  width: number;
+  height: number;
+  deviceScaleFactor: number;
+  disabled: boolean;
+} | null;
```

Comment:
Should probably not have `| null` right here. Try to avoid using `null` overall, but maybe you can have it as `null` on line 111 at the usage site below if that's a request payload where it makes sense, and adding a `| undefined` in the `formFactorToScreenEmulationMap` further down.